<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFace</class>
 <widget class="QMainWindow" name="MainFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>632</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>996</width>
    <height>632</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>996</width>
    <height>632</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Classification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Classification/Resources/bg.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListView" name="listView">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTableView" name="tableView">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionView"/>
   <addaction name="actionClass"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="classification.qrc">
     <normaloff>:/Classification/Resources/open.png</normaloff>:/Classification/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="classification.qrc">
     <normaloff>:/Classification/Resources/save.png</normaloff>:/Classification/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionView">
   <property name="icon">
    <iconset resource="classification.qrc">
     <normaloff>:/Classification/Resources/view.png</normaloff>:/Classification/Resources/view.png</iconset>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View</string>
   </property>
  </action>
  <action name="actionClass">
   <property name="icon">
    <iconset resource="classification.qrc">
     <normaloff>:/Classification/Resources/classes.png</normaloff>:/Classification/Resources/classes.png</iconset>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
   <property name="toolTip">
    <string>Class</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="classification.qrc"/>
 </resources>
 <connections/>
</ui>
